mailpoet/assets/js/src/newsletters/send/sender-address-field.jsx

```diff
@@ -80,12 +80,16 @@ class SenderField extends Component {
 
     if (!isFieldValid(this.domElementSelector)) {
       validateField(this.domElementSelector);
+      // Hide domain error to not stack error messages
+      this.showSenderDomainError(false);
```

Comment:
I haven't tested it, but just looking at the code, is this doing what you want? It looks like `showSenderDomainError` can only ever set the state to true.

```js
  showSenderDomainError = (status) => {
    if (!status) return;

    this.setState({ showSenderDomainWarning: true });
  };
```

Reply:
Fixed! Thanks for catching that 
![image](https://github.com/mailpoet/mailpoet/assets/8002881/b9559182-63a8-4c93-aa7b-e0641400e146)